### PR TITLE
Delete motor

### LINKED ISSUE #7: [xpt 2.0]Additional features frontend
- When the user clicks the "Delete item" link in the navigation panel they can see a list of all items with title and "Delete" button.

- When the user clicks the "Delete" button, the selected item is marked as removed and does not show on the main list anymore.

### PR DESCRIPTION
**Pull request by rahimialijan**
# Delete Motor 
In this part of my project, I have the following tasks

1. Add delete motor component
2. Add style for delete page
3. Add responsive style for add motor page
4. Add reducer function for delete motor
5. Clean the linter error
6. Work on responsiveness of some pages

### CODE REVIEW

File: src/components/DeleteMotor/DeleteMotor.jsx [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
+            loopFillGroupWithBlank
+            slidesPerView={3}
+            spaceBetween={20}
+            slidesPerGroup={1}
+          >
+            {motorData.map((item) => (
+              <SwiperSlide key={item.id}>
+                <img
+                  src={item.photo}
+                  alt={item.brand_name}
```
Comment (binodbhusal) [inline src/components/DeleteMotor/DeleteMotor.jsx]: As per project requirements, When the user clicks the "Delete item" link in the navigation panel they can see a list of all items with title and "Delete" button. So please kindly make changes to list all motors.
Please also remove unnecessary items such as social media icons.

**Review by binodbhusal — CHANGES_REQUESTED** [review]
Hi @rahimialijan  ,
Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

**Highlights**

 No linter errors
 Descriptive PR
**Required Changes ♻️**
Please Check the comments under the review.
Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/components/DeleteMotor/DeleteMotor.jsx (hunk 1 above)
Comment (rahimialijan) [reply src/components/DeleteMotor/DeleteMotor.jsx]: done

**Review by Tshobohwa — APPROVED** [review]
# STATUS: APPROVED 🎉 

Hi @rahimialijan ,
You have done a good job updating this required changes.
There is nothing more to say than "...Time to merge it" :shipit:



Cheers and happy coding 🎉

**Pull request merged**